### ISSUE TITLE
Unable to import Keras model, error in hdf5 file

### ISSUE THREAD

**Issue by Nickedude** [deeplearning4j/deeplearning4j#4891]
#### Issue Description

Hi, 

I haven't been able to find a post about this problem before, which is why I'm posting this now. I've created a model in Keras that i would like to import into dl4j. However, dl4j seems to be unable to read the hdf5 file. I've tried exporting the model as a whole to an hdf5 file and I've tried exporting the configuration to a json file along with the weights to hdf5 file. In both cases dl4j seems to be unable to read the hd5j file. The java code I'm trying to execute is simply the following:

MultiLayerNetwork model = KerasModelImport.importKerasSequentialModelAndWeights(path);

I receive the following error:

SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
HDF5-DIAG: Error detected in HDF5 (1.10.0-patch1) thread 0:
  #000: H5G.c line 474 in H5Gopen2(): unable to open group
    major: Symbol table
    minor: Can't open object
  #001: H5Gint.c line 289 in H5G__open_name(): group not found
    major: Symbol table
    minor: Object not found
  #002: H5Gloc.c line 430 in H5G_loc_find(): can't find object
    major: Symbol table
    minor: Object not found
  #003: H5Gtraverse.c line 869 in H5G_traverse(): internal path traversal failed
    major: Symbol table
    minor: Object not found
  #004: H5Gtraverse.c line 641 in H5G_traverse_real(): traversal operator failed
    major: Symbol table
    minor: Callback failed
  #005: H5Gloc.c line 385 in H5G_loc_find_cb(): object 'dense_1_Wdense_1_b' doesn't exist
    major: Symbol table
    minor: Object not found
Exception in thread "main" java.lang.RuntimeException: Unknown exception.
	at org.bytedeco.javacpp.hdf5$CommonFG.openGroup(Native Method)
	at org.deeplearning4j.nn.modelimport.keras.Hdf5Archive.getDataSets(Hdf5Archive.java:140)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.helperImportWeights(KerasModel.java:360)
	at org.deeplearning4j.nn.modelimport.keras.KerasSequentialModel.<init>(KerasSequentialModel.java:133)
	at org.deeplearning4j.nn.modelimport.keras.KerasSequentialModel.<init>(KerasSequentialModel.java:56)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel$ModelBuilder.buildSequential(KerasModel.java:646)
	at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasSequentialModelAndWeights(KerasModelImport.java:156)

As far as i can see the hdf5 file seems fine, but I might be wrong. I've tried exporting it several times with several different versions of Keras and I receive the same result. 
The hdf5 file can be found here: https://ufile.io/haqjs

The ANN configuration is pretty straight forward. The code in python is as follows:

import keras
from keras.models import Sequential
from keras.layers import Dense        
classifier = Sequential()
classifier.add(Dense(output_dim = 39, activation = 'relu', input_dim = 78)) 
classifier.add(Dense(output_dim =39, activation = 'relu'))
classifier.add(Dense(output_dim =39, activation = 'relu'))
classifier.add(Dense(output_dim = 1, activation = 'sigmoid'))
classifier.compile(optimizer = 'adam', loss = 'binary_crossentropy', metrics = ['accuracy'])
classifier.fit(X_train, y_train, batch_size = 10, nb_epoch = 100)

#### Version Information
* Deeplearning4j version = 0.9.1
* OS = Ubuntu 16.04
* Keras version = 1.2.2

Not using CUDA nor GPU. 

I would be very thankful for any help.
